frontend/src/libs/enums/loader.enum.ts

```diff
@@ -0,0 +1,6 @@
+const ILoader = {
```

Comment:
Or LoaderType, or everything else, not `ILoader`
```suggestion
const LoaderShape = {
```